Makefile

```diff
@@ -443,16 +516,149 @@ doc_diagram_sources+=$(addprefix src/lib/transition_frontier/res/,*.dot *.tex *.
 .PHONY: doc_diagrams
 doc_diagrams: $(addsuffix .png,$(wildcard $(doc_diagram_sources))) ## Generate documentation diagrams
 
+.PHONY: export_git_env_vars
+export_git_env_vars: ## Export git environment variables for use in scripts
+	KEEP_MY_TAGS_INTACT=true \
+		./scripts/export-git-env-vars.sh
+
+########################################
+# Debian packages
+
+# Helper function for building Debian packages
+define build_debian_package
```

Comment:
For history: https://www.gnu.org/software/make/manual/html_node/Canned-Recipes.html#Canned-Recipes.